nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/tools/ScannerTool.java

```diff
@@ -650,4 +663,18 @@ void showTRACTGUI() {
         tractGUI.showMCplot();
     }
 
+    /**
+     * Disable the options if the processor is busy processing.
+     * @param newStatus
+     */
+    public void processorAvailableStatusUpdated(Boolean newStatus) {
```

Comment:
This would be better as `boolean` as I don't think there's ever a need for `newStatus` to be `null`. Also `setDisable()` takes in a boolean

Reply:
was a copy paste typo, fixed it here and the other two spots it should have been `boolean`